backend/src/main/java/com/htecgroup/skynest/controller/BucketController.java

```diff
@@ -288,13 +288,13 @@ public List<BucketResponse> getBuckets() {
                   })
             })
       })
-  @PutMapping("delete/{uuid}")
-  public ResponseEntity<Boolean> deleteBucket(@PathVariable UUID uuid) {
-    bucketService.deleteBucket(uuid);
+  @PutMapping("/{bucketId}/delete")
```

Comment:
Since you are changing this, I would change this to be Delete http, for consisted level around the app